CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
```

Comment:
```suggestion
We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.
```
I suggest to move this to the top, so it starts on a more positive note

Reply:
No objection to that.